components/widgets/tour_tip/tour_tip.tsx

```diff
@@ -206,6 +208,7 @@ const TourTip = ({
             {show && (
                 <Tippy
                     showOnCreate={show}
+                    onHidden={onHidden}
```

Comment:
Check if above solution works for you, we mostly will be replacing tippy with popper/floating UI so trying to expose minimum of tippy api, if not please ignore above comment but I think that should work I am doing something similar.